contact2.php

```diff
@@ -51,10 +53,10 @@
    }
    // http://php.net/manual/en/function.mail.php
    $toaddress = $contact_to;
-   $subject = 'Contact from Website';
-   $mailcontent = "Contact name: $name \n\n
-   			       Contact email: $email \n\n
-                   Contact comments: $message \n";
+   // The spaces in this string get transfer to the e-mail, so be careful when adding/removing spaces here.
+   $mailcontent = "   Contact name: $name,
```

Comment:
Remove spaces before contact name
```suggestion
   $mailcontent = "Contact name: $name,
```